data_ingestion/04_Ingest_data_with_EMR.ipynb

```diff
@@ -0,0 +1,237 @@
+{
```

Comment:
<p>Reword</p><blockquote>Now that we read in the data, from here you can pre-process the data with Spark in EMR cluster, build ML pipeline, and train models in scale.</blockquote><p>To </p><blockquote>Now that you have read in the data, you can pre-process the data with Spark in an EMR cluster, build an ML pipeline, and train models in scale.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1602/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>